packages/frontend/src/main/pages/auth/TheRegistration.vue

```diff
@@ -296,19 +301,31 @@ export default {
     }
   },
   methods: {
-    debouncedPwdTest: debounce(async function () {
+    async validatePasswordStrength() {
```

Comment:
I think there was reason to debounce this test. Is that not needed anymore?

Reply:
@gjedlicska it didn't work very well, it was often delayed and just confused the user. from now on the check is invoked once you actually press the submit button so that it's clear that the message that appears relates to the last submit. so no, it's not necessary anymore

Reply:
blah, makes sense, should have read more carefully